pkg/fakerp/clusterstatus.go

```diff
@@ -0,0 +1,45 @@
+package fakerp
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/plugin"
+)
+
+// handleGetControlPlanePods handles admin requests for the list of control plane pods
+func (s *Server) handleGetControlPlanePods(w http.ResponseWriter, req *http.Request) {
```

Comment:
Either rebase this code on top of your other PR to take advantage of the refactoring, or open a follow-up issue to dry this function.

Reply:
rebased